src/qml/DashBoard.qml

```diff
@@ -45,7 +44,7 @@ Controls.Pane {
   onShowMenu: mainMenu.__popup(Qt.rect(menuButton.x + 2 * menuButton.width, menuButton.y, mainMenu.__popupGeometry.width, mainMenu.__popupGeometry.height), 0, 0)
 
   onCurrentLayerChanged: {
-    if ( currentLayer.readOnly && stateMachine.state == "digitize" )
+    if ( currentLayer && currentLayer.readOnly && stateMachine.state == "digitize" )
```

Comment:
I suppose if you did it, then it's needed. But why?

Reply:
isn't it to avoid complaints that the layer is read-only when there is no layer selected?

Reply:
So apparently readOnly of "no layer" is true? I'm just curious...

Reply:
It's because there is a warning that it cannot access the property readOnly when undefined... Fine. It's clear now.